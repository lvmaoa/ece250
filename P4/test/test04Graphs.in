i 1 2 1
i 1 3 1
i 1 4 1
i 2 3 1
i 4 2 1
p 2
p 1
d 2
p 2
p 1
p 4
i 2 3 1
i 4 2 1
i 1 2 1
p 1
p 2
p 4
exit
