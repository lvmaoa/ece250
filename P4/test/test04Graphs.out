success
success
success
success
success
3 
2 3 4 
success

3 4 

success
success
success
3 4 2 
3 
2 
